src/services/mongoose/wallets.ts

```diff
@@ -36,28 +35,12 @@ export const WalletsRepository = (): IWalletsRepository => {
     if (account instanceof Error) return account
     try {
       // FLASH FORK: create IBEX account if currency is USD
-      let ibexAccountId = ""
-      if (currency === "USD") {
-        const IbexAccountCreationResponse = await requestIBexPlugin(
-          "POST",
-          IbexRoutes.API_CreateAccount,
-          {},
-          {
-            name: accountId,
-            currencyId: 3,
-          },
-        )
-
-        if (
-          !IbexAccountCreationResponse ||
-          !IbexAccountCreationResponse.data ||
-          !IbexAccountCreationResponse.data["data"]["id"]
-        ) {
-          console.error({ error: "unable to get IbexAccountCreationResponse" })
-        } else {
-          ibexAccountId = IbexAccountCreationResponse.data["data"]["id"]
-        }
-      }
+      const resp = await Ibex.createAccount({
```

Comment:
Does this need an if statement or something so it only makes a new account on IBEX when the currency === 'USD' ?

Reply:
this seems to work
```

let ibexAccountId
    try {
      // FLASH FORK: create IBEX account if currency is USD
      if (currency === "USD") {
        const resp = await Ibex.createAccount({
          name: accountId,
          currencyId: 3,
        })
        if (resp instanceof IbexEventError) return resp
        ibexAccountId = resp.id
      }
      const wallet = new Wallet({
        _accountId: toObjectId<AccountId>(accountId),
        id: ibexAccountId || crypto.randomUUID(),
```